CONTRIBUTING.md

```diff
@@ -0,0 +1,199 @@
+# Contributing to AutoEmulate
+
+💫⚙️🤖 We're excited that you're here and want to contribute. 🤖⚙️💫
+
+By joining our efforts, you will be helping to <INSERT ONE-LINER ABOUT THE PURPOSE OF THE PACKAGE>.
+
+We want to ensure that every user and contributor feels welcome, included and supported to
+participate in the AutoEmulate community. Whether you're a seasoned developer, a machine learning
+researcher, a data scientist, or just someone eager to learn and contribute, **you are
+welcome here**. We value every contribution, be it big or small, and we appreciate the unique
+perspectives you bring to the project.
+
+We hope that the information provided in this document will make it as easy as possible for you to
+get involved. If you find that you have questions that are not discussed below, please let us know
+through one of the many ways to [get in touch](#get-in-touch).
+
+## Important Resources
+
+If you'd like to find out more about AutoEmulate, make sure to check out:
+
+1. **README**: For a high-level overview of the project, please refer to our README.
+2. **Documentation**: For more detailed information about the project, please refer to
+   our [documentation](<LINK TO DOCUMENTATION HERE>).
+3. **Project Roadmap**: Familiarize yourself with our direction and goals by checking
+   out [the project's roadmap](<LINK TO ROADMAP HERE>).
+
+## Get in touch
+
+The easiest way to get involved with the active development of AutoEmulate is to join our regular
+community calls. The community calls are currently on a hiatus but if you are interested in
+participating in the forthcoming community calls, which will start in 2024, you should join our
+Slack workspace, where conversation about when to hold the community calls in the future will take
+place.
+
+**Slack Workspace**: Join
+our [AutoEmulate Slack channel](<LINK TO SIGN-UP OR TO THE SLACK TEAM>) for
```

Comment:
Yeah - I have run into, many times now, the problem that we cannot just add people to the Turing slack, but might need to set up a separate Slack workspace or something. We'll wait and see -- I think it might be a point to make something larger (like a TRIC workspace) and then take it from there.. bigger questions, so let's hold off on it for now!